WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -126,4 +126,10 @@ def wca_date_range(from_date, to_date, options={})
     options[:separator] = '-'
     date_range(from_date, to_date, options)
   end
+
+  def alert(type, content=nil, note: false, &block)
+    content = capture(&block) if block_given?
+    content.prepend("<strong>Note:</strong> ") if note
```

Comment:
That's right, I didn't do this before because I called `.html_safe` on the whole content.